src/evict/evict_page.c

```diff
@@ -266,20 +283,25 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
 
         WT_STAT_CONN_DATA_INCR(session, cache_eviction_fail);
     }
+    if (!session->evict_timeline.reentry_hs_eviction) {
+        eviction_time_milliseconds = eviction_time / WT_THOUSAND;
+        if (eviction_time_milliseconds > conn->cache->evict_max_ms)
+            conn->cache->evict_max_ms = eviction_time_milliseconds;
+        if (eviction_time_milliseconds > 60000)
```

Comment:
I think `WT_MINUTE * WT_THOUSAND` would be the more coding standard consistent way to phrase this. See generation.c:169.

Reply:
Yes! @pmacko86 did a ticket when he started to get rid of large hard-coded constants. We do not want to re-introduce them.

Reply:
done